dojo/problem/views.py

```diff
@@ -100,12 +112,24 @@ def get(self, request: HttpRequest):
 class ListOpenProblems(ListProblems):
     filter_name = "Open"
 
-    def get_problems(self, request: HttpRequest):
+    def get_problems(self, request: HttpRequest, products=None):
+        finding_ids_in_problems = [
+            fid for p in self.problems_map.values() for fid in p.finding_ids
+        ]
+        if products:
+            findings = Finding.objects.filter(
+                id__in=finding_ids_in_problems,
+                test__engagement__product__in=products,
+                active=True,
+            )
+        else:
+            findings = Finding.objects.filter(
+                id__in=finding_ids_in_problems,
+                active=True,
+            )
```

Comment:
It seems weird t me that this function uses `Finding.objects.filter` while `ListProblems.get_problems` uses a list comprehension. Why don't we make these functions the same, but pass `active=False` by default in `ListProblems.get_problems`, and here we can just `super(request, products, True)` to call the existing function. This way we would avoid code duplication.